### PR TITLE
Add comments

### LINKED ISSUE #8: [4pt]  - Create feature: add new comment - Student B
Implement the "add new comment" form on the Comments pop up according to the wireframe:

Screenshot 2021-06-27 at 22 32 36

When the user clicks on the "Comment" button, the data is recorded in the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) and the screen is updated.

### PR DESCRIPTION
**Pull request by aimalamiri**
## The user now can comment about meals

1. Design comments form
2. The user can create a comment
3. The comments list and counter will be updated when the user adds a new comment.

closes #8

### CODE REVIEW

File: src/modules/Modal.js [hunk 1]
```diff
@@ -12,32 +11,15 @@ export default class Modal {
       this.element.querySelector(`#modal-${selector}`)[attr ?? 'textContent'] = this.data[value];
     };
 
+    this.element.querySelector('#modal-ingredients').innerHTML += '';
```
Comment (rbanos-mv) [inline src/modules/Modal.js:14]: Are you sure you wan to `append` an empty string? Maybe you want to `assign` an empty string.

File: src/modules/Modal.js [hunk 2]
```diff
@@ -33,11 +33,13 @@ export default class Modal {
...
-      await api.addComment(this.data.idMeal, { username, comment: message });
-      commentForm.reset();
-      this.#loadComments();
+      const username = this.element.querySelector('#modal-input-name').value || '';
+      const message = this.element.querySelector('#modal-input-comment').value || '';
+      if (username.trim() !== '' && message.trim() !== '') {
+        await api.addComment(this.data.idMeal, { username, comment: message });
+        commentForm.reset();
+        this.#loadComments();
+      }
```
Comment (rbanos-mv) [inline src/modules/Modal.js]: Please consider the following:
- if username or message have spaces at the beginning or at the end, they will be trimmed for the if condition, but when you add the comment to the API, the spaces will be sent.
- Maybe you want to trim spaces before sending to the API.

**Review by rbanos-mv — APPROVED** [review]
Well done! Congratulations. Before you merge, please consider two observations. I was not sure what you intended to achieve in these cases. If you consider there's no need to make any change, proceed to merge.

File: src/modules/Modal.js (hunk 2 above)
Comment (aimalamiri) [reply src/modules/Modal.js]: Yes, you're right, and I fixed this issue.

File: src/modules/Modal.js (hunk 1 above)
Comment (aimalamiri) [reply src/modules/Modal.js:14]: Yes, I'm doing this because I need to clear the previous ingredients from the list.

**Pull request merged**